GoMeet/src/main/webapp/js/meeting-time-dao.js

```diff
@@ -38,9 +21,10 @@ class MeetingTimeDAO {
       throw new Error(INVALID_PARAM_TYPE);
     }
 
-    let urlString = MeetingTimeDAO.url({'meetingTimeId': meetingTimeId});
-    let results = await fetch(urlString).then((results) => results.json());
-    return results;
+    let urlString = DAOUtils.url(MeetingTimeDAO.endpoint, {'meetingTimeId': meetingTimeId});
+    let results = await fetch(urlString);
+    let resultsJson = await results.json();
```

Comment:
Why did you change this from the arrow function style from before?

Reply:
I think when I rebased, commits were moved around, so the refactoring that I did was over written. I've written history, and changed it back now :)